### PR TITLE
Get data from API to single scientific work page

### PR DESCRIPTION
**Pull request by JustD4nTe**
- Handle adding reviews/answers to review
- Get data to display information about work (with reviews and answers)
- Add ability to download review (only when reviewer upload it as file)
- Refactor components in the page:
  - Rename components
  - Implement using flex display (almost in all components on the page)
- Separate reviews for users:
  - normal user cannot see reviews
  - reviewer can see only own reviews and answers to it
  - author of work can see all reviews

### CODE REVIEW

**Review by KamilDonda — APPROVED** [review]

**Review by upsu90 — CHANGES_REQUESTED** [review]
when an author doesn't have a photo, I have an error

**Review by Kwolik — CHANGES_REQUESTED** [review]
- Download full work after entering a specific job, it does not work,
- A reviewer, when he is not assigned to any work, cannot view it
https://media.discordapp.net/attachments/765345766335578172/799577181213818900/unknown.png?width=960&height=446
https://media.discordapp.net/attachments/765345766335578172/799576250938032129/unknown.png?width=960&height=446

File: Kongres.Api/Kongres.Api.Application/Services/ReviewService.cs [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                Reviewer = reviewer,
+                DateReview = DateTime.Now,
+                Rating = rating,
+            };
+
+            // reviewer can add review only one way (string or file, not both!)
+            if (reviewMsg is null)
+                review.File = await _fileManager.SaveFileAsync(reviewFile);
+            else if (reviewFile is null)
+                review.Comment = reviewMsg;
```
Comment (Kwolik) [inline Kongres.Api/Kongres.Api.Application/Services/ReviewService.cs]: The assumption was that both versions of the review could be added.

**Review by Kwolik — CHANGES_REQUESTED** [review]

**Review by Kwolik — APPROVED** [review]
Finally. Good job

**Review by upsu90 — CHANGES_REQUESTED** [review]
We should also be able to add other characters :c 
image

**Review by upsu90 — APPROVED** [review]

**Pull request merged**